### DISCUSSION TITLE
Random 'Resource not accessible by integration' Errors

### DISCUSSION THREAD

**Discussion by emmyoop** [community/community#60784] [Actions]
### Select Topic Area

Bug

### Body

I recently noticed intermittent 403 errors.

```
RequestError [HttpError]: Resource not accessible by integration
```

on at least two workflows we've been running for a while.  Example failures [here](https://github.com/dbt-labs/dbt-core/actions/runs/5523528705/jobs/10074580966?pr=8069) and [here](https://github.com/dbt-labs/dbt-redshift/actions/runs/5544736066/jobs/10122634545).

Both of these workflows have permissions explicitly defined and run without this error both before and after we get these errors.  Both of them are also failing when using marketplace actions.  I'm unable to determine why this would be happening.  

Has anyone else experienced this or have ideas on how to dig in to the cause?

**Comment by IanButterworth** [comment]
The julia language TagBot has long been suffering from these unexplained sporadic errors, sometimes permanent for a repo without any changes. https://github.com/JuliaRegistries/TagBotErrorReports/issues/83

Is any more understood about this since this was posted?
